src/LuaRenderer/nodes/statements/renderMethodDeclaration.ts

```diff
@@ -5,9 +5,8 @@ import { renderStatements } from "LuaRenderer/util/renderStatements";
 
 export function renderMethodDeclaration(state: RenderState, node: lua.MethodDeclaration) {
 	let result = "";
-	result +=
-		state.indent + `function ${render(state, node.expression)}:${node.name}(${renderParameters(state, node)})\n`;
+	result += state.line(`function ${render(state, node.expression)}:${node.name}(${renderParameters(state, node)})`);
```

Comment:
Very satisfying that this turned into one line!

Reply:
Yep, looks cleaner ;)